ompi/mpi/c/Makefile.am

```diff
@@ -103,7 +105,6 @@ libmpi_c_mpi_la_SOURCES = \
         cart_sub.c \
         close_port.c \
         comm_accept.c \
-        comm_c2f.c \
```

Comment:
i initially this was a prerequisite, and if not, then the second commit cannot compile if configure'd with --disable-mpi-fortran
on second thought, this was a major overkill and i can get everything working by simply
#if OMPI_BUILD_FORTRAN_BINDINGS
protect ompi/mpi/fortran/base/constants.h